starport/pkg/gomodule/gomodule.go

```diff
@@ -32,6 +37,21 @@ func ParseAt(path string) (*modfile.File, error) {
 	return modfile.Parse("", gomod, nil)
 }
 
+// ValidateGoModule check if the cosmos-sdk and the tendermint packages are imported.
+func ValidateGoModule(module *modfile.File) error {
```

Comment:
I think we need to move this func somewhere else because it's sdk specific but pkg/gomodule is a generic pkg.